.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: swift
+osx_image: xcode11.2
+
+before_install:
+  - carthage update
```

Comment:
This has been broken by my SPM update and needs to be resolved. Sorry about that @vexy